### DISCUSSION TITLE
Profile Achievments dissapered

### DISCUSSION THREAD

**Discussion by ELHart05** [community/community#122716] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I had 5 achivments in my github profile which are:

- Public sponsor
- Pull shark x2
- Quickdraw
- Pair Extraordinaire
- YOLO

Suddenly I checked my profile to find only 3 as they're now,
It's just weird especially the Pull shark decrementing to one as I have more than 16 PR (60 most of them are accepted 
![image](https://github.com/community/community/assets/96151694/5814dcb6-9f73-4bd2-b83c-2a364123daec)

**Comment by davevad93** [comment]
Hi @ELHart05 ,

Try to take a look at a couple of things:

1. First try checking in your profile settings if achievements are hidden, it happened to me once even though I didn't hide anything.
2. If that's isn't the case try going to the `Achievements` tab on your profile, click on each achievement card, on the top left side you'll see an eye icon, click on `Hide from profile` and then click again on `Show on profile` . This method fixed my `Pull Shark` bronze achievement which wasn't displayed in the profile tab.

**Reply by ELHart05** [reply]
Going to settings actually helped!

**Comment by ELHart05** [comment]
I solved the issue by going to profile and enabling the Include private contributions on my profile (which was unchecked even thought I didn't change it at all)
![image](https://github.com/community/community/assets/96151694/c4b9c045-b1d2-4005-99fa-494ac73444ac)
